sellotape/sellotape_dj/chat/templates/chat/index.html

```diff
@@ -0,0 +1,24 @@
+<html>
```

Comment:
Is there a specific reason not to make use of the `base.html` template you guys implemented?

Reply:
It will be ported to the template in a later PR. the 'base.html' wasn't the real-deal, we now have a real template and will add it.